client.go

```diff
@@ -349,7 +362,7 @@ func (c *ovndb) reconnect() {
 		log.Printf("%s disconnected. Reconnecting ... \n", c.addr)
 		retry := 0
 		for range ticker.C {
-			if err := connect(c); err != nil {
+			if err := connect(c, true); err != nil {
```

Comment:
I think you just forgot to remove the new arg here in the revision.

Reply:
Yep, sorry. Re-pushed